java/app/grapheneos/setupwizard/action/MigrationActions.kt

```diff
@@ -15,7 +17,28 @@ object MigrationActions {
     fun launchMigration(context: SetupWizardActivity) {
         Log.d(TAG, "launchMigration")
         val intent = Intent(ACTION_SEEDVAULT_RESTORE)
-        SetupWizard.startActivityForResult(context, intent)
+        try {
+            SetupWizard.startActivityForResult(context, intent)
```

Comment:
Hi thanks for the PR @jf916 .
my suggestion would be to do this inside `MigrationActivity` (as early in the activity initialization phase as possible).
we could try to resolve the intent action to see if leads anywhere, if not then it would be better not to show any related UI to the user and just skip that step.

Reply:
@jiteshsingh I have moved the check to MigrationActivity

Reply:
>  if not then it would be better not to show any related UI to the user and just skip that step.

^ pending

Reply:
done

Reply:
@jiteshsingh see comment on line 27 of MigrationActivity.java